### PR TITLE
Optimize model preview/thumbnail icon loading and rename `_BACKGROUND` enumerators to `_NULL`

### PR DESCRIPTION
**Pull request by guruathwal**
- Optimize model preview/thumbnail icon loading. The G-code file is now open only once instead of reopening it every time for all the enabled preview types.
-  Rename all the enumerators ending with `_BACKGROUND` to `_NULL' which are used to define null/empty elements.

### CODE REVIEW

File: TFT/src/User/Menu/StatusScreen.c [hunk 1]
```diff
@@ -25,32 +25,32 @@ const MENUITEMS StatusItems = {
   LABEL_READY,
   // icon                          label
   {
-    {ICON_STATUS_NOZZLE,           LABEL_BACKGROUND},
-    {ICON_STATUS_BED,              LABEL_BACKGROUND},
-    {ICON_STATUS_FAN,              LABEL_BACKGROUND},
-    {ICON_STATUS_SPEED,            LABEL_BACKGROUND},
+    {ICON_STATUS_NOZZLE,           LABEL_NULL},
+    {ICON_STATUS_BED,                    LABEL_NULL},
```
Comment (digant73) [inline TFT/src/User/Menu/StatusScreen.c:29]: wrong indent in these two lines

File: TFT/src/User/Menu/StatusScreen.c [hunk 2]
```diff
@@ -25,32 +25,32 @@ const MENUITEMS StatusItems = {
...
+    {ICON_NULL,                    LABEL_NULL},
     {ICON_PRINT,                   LABEL_PRINT},
   }
 };
 
 const ITEM BedItems[2] = {
   // icon                        label
-  {ICON_STATUS_BED,              LABEL_BACKGROUND},
-  {ICON_STATUS_CHAMBER,          LABEL_BACKGROUND},
+  {ICON_STATUS_BED,                    LABEL_NULL},
```
Comment (digant73) [inline TFT/src/User/Menu/StatusScreen.c:46]: wrong indent here

File: TFT/src/User/Configuration.h [hunk 3]
```diff
@@ -1346,10 +1346,10 @@
  *                    flexible but requires a dedicated post-processing of gcode files for
  *                    most slicers. "Classic" is used as fallback.
  *     Base64 PNG:    A specific thumbnail comment identifies the location of a Base64-encoded
- *                    PNG thumbnail. It is slower as classic but most flexible. It does _not_
+ *                    PNG thumbnail. It is slower as classic but most flexible. It does not
  *                    require dedicated post-processing of gcode files for most slicers.
  *                    "RGB565 bitmap" and "Classic" are used as fallback.
  */
-#define THUMBNAIL_PARSER 0  // Default: 0
+#define THUMBNAIL_PARSER 2  // Default: 0
```
Comment (digant73) [inline TFT/src/User/Configuration.h:1353]: did you enable it by mistake?

Comment (guruathwal) [reply TFT/src/User/Configuration.h:1353]: yes

**Pull request merged**